### PR TITLE
SG-24400 Remove default config from the advanced setup

### PR DESCRIPTION
**Pull request by NorberMV**
Removed the "default_config_page.py" file to eliminate the project Legacy Default configuration and enabled the Default configuration.

### CODE REVIEW

File: python/setup_project/setup_type_page.py [hunk 1]
```diff
@@ -53,15 +54,18 @@ def set_disk_page(self, page):
...
-        self._default_configs_page_id = page.page_id()
+        self._storage_map_page_id = page.page_id()
 
     def nextId(self):
         # return the appropriate id for the current selection
         selection = self._config_type_button_group.checkedId()
-        if (selection == 0) and self._default_configs_page_id is not None:
-            return self._default_configs_page_id
+
+        if (selection == 0) and self._storage_map_page_id is not None:
```
Comment (000paradox000) [inline python/setup_project/setup_type_page.py:65]: Have you consider removing ( ) in this line?

File: python/setup_project/setup_type_page.py [hunk 2]
```diff
@@ -53,15 +54,18 @@ def set_disk_page(self, page):
...
+            wiz = self.wizard()
+            wiz.set_config_default()  # Function to validate the uri
+            return self._storage_map_page_id
         if (selection == 1) and self._project_page_id is not None:
```
Comment (000paradox000) [inline python/setup_project/setup_type_page.py:69]: Same note here

File: python/setup_project/setup_type_page.py [hunk 3]
```diff
@@ -53,15 +54,18 @@ def set_disk_page(self, page):
...
             return self._project_page_id
         elif (selection == 2) and self._github_page_id is not None:
```
Comment (000paradox000) [inline python/setup_project/setup_type_page.py:71]: Same note here

**Review by 000paradox000 — CHANGES_REQUESTED** [review]

File: python/ui/setup_project.py [hunk 4]
```diff
@@ -99,66 +99,6 @@ def setupUi(self, Wizard):
         self.verticalLayout.addItem(spacerItem4)
         self.verticalLayout.setStretch(13, 1)
         Wizard.addPage(self.setup_type_page)
-        self.default_configs_page = DefaultConfigPage()
```
Comment (jfboismenu) [inline python/ui/setup_project.py:102]: This change should be accompanied with an update to the resources folder. [setup_project.py is auto-generated](https://github.com/shotgunsoftware/tk-framework-adminui/blob/master/resources/build_resources.sh#L42).

**Review by jfboismenu — CHANGES_REQUESTED** [review]

**Review by mathurf — APPROVED** [review]
LGTM!

Tested it locally and all issues found are now fixed.
Ready to merge if Ariel is fine with the code. :) 

Great work!

**Review by 000paradox000 — APPROVED** [review]
LGTM very good job

**Pull request merged**